intro/content/intro.tex

```diff
@@ -84,28 +89,27 @@ \section{Ablauf}
 
 \begin{frame}{Ablauf}
   \begin{description}
-    \item[Montag] Programmieren mit Python / Erstellen von Plots
+    \item[Montag] Programmieren mit Python
+    \item[Dienstag] Erstellen von Plots / Auswerten
       \begin{itemize}
-        \item Python
         \item NumPy
         \item matplotlib
       \end{itemize}
-    \item[Dienstag] Erstellen von Plots / Auswerten / Fehlerrechnung
+    \item[Mittwoch] Auswerten / Fehlerrechnung
       \begin{itemize}
-        \item matplotlib
         \item scipy
         \item uncertainties
       \end{itemize}
-    \item[Mittwoch] Kommandozeile und Automatisierung
+    \item[Donnerstag] Kommandozeile und Automatisierung
```

Comment:
should be changed on the homepage too

Reply:
Yes, but that's a different repo

Reply:
Correct, then I can change it there as well.

Reply:
So, approve?